pkg/nsx/services/common/types.go

```diff
@@ -48,8 +48,8 @@ const (
 	TagScopeNSXServiceAccountCRUID     string = "nsx-op/nsx_service_account_uid"
 	TagScopeNSXProjectID               string = "nsx-op/nsx_project_id"
 	TagScopeProjectGroupShared         string = "nsx-op/is_nsx_project_shared"
-	TagScopeVPCCRName                  string = "nsx-op/vpc_name"
-	TagScopeVPCCRUID                   string = "nsx-op/vpc_uid"
+	TagScopeNetworkInfoCRName          string = "nsx-op/vpc_name"
```

Comment:
maybe we need to change the tag too, this tag is on nsx vpc object recording vpc(now should be networkinfo) cr uid and name.